static/app/components/alert.tsx

```diff
@@ -104,92 +172,67 @@ const alertStyles = ({
   `;
 };
 
-const StyledTextBlock = styled('span')`
-  line-height: ${p => p.theme.text.lineHeightBody};
-  position: relative;
-  flex: 1;
+const Wrap = styled('div')<AlertProps & {hovered: boolean}>`
+  ${alertStyles}
 `;
 
-const MessageContainer = styled('div')`
+const IconWrapper = styled('div')`
   display: flex;
+  height: calc(${p => p.theme.fontSizeMedium} * ${p => p.theme.text.lineHeightBody});
+  padding-top: ${space(1.5)};
+  padding-bottom: ${space(1.5)};
+  padding-left: ${space(2)};
+  padding-right: ${space(1)};
```

Comment:
```suggestion
  padding: ${space(1.5)} ${space(1)} ${space(1.5)} ${space(2)};
```

🤷 nit

Reply:
most people know the top right bottom left rule

Reply:
done! https://github.com/getsentry/sentry/pull/33356/commits/de147ffcff15d4b8abc45ea4708984fed8c20f9b